specification/app/resource-manager/Microsoft.App/preview/2023-04-01-preview/Jobs.json

```diff
@@ -0,0 +1,1193 @@
+            }
+          }
+        }
+      }
+    },
+    "JobsCollection": {
+      "description": "Container Apps Jobs collection ARM resource.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Job"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "RegistryCredentials": {
+      "description": "Container App Private Registry",
+      "type": "object",
+      "properties": {
+        "server": {
+          "description": "Container Registry Server",
+          "type": "string"
+        },
+        "username": {
+          "description": "Container Registry Username",
+          "type": "string"
+        },
+        "passwordSecretRef": {
+          "description": "The name of the Secret that contains the registry login password",
+          "type": "string"
+        },
+        "identity": {
+          "description": "A Managed Identity to use to authenticate with Azure Container Registry. For user-assigned identities, use the full user-assigned identity Resource ID. For system-assigned identities, use 'system'",
+          "type": "string"
+        }
+      }
+    },
+    "JobSecretsCollection": {
+      "description": "Container Apps Job Secrets Collection ARM resource.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "./CommonDefinitions.json#/definitions/Secret"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        }
+      }
+    },
+    "ReplicaCompletionCount": {
+      "description": "Minimum number of successful replica completions before overall job completion.",
+      "format": "int32",
+      "type": "integer"
+    },
+    "Parallelism": {
+      "description": "Number of parallel replicas of a job that can run at a given time.",
+      "format": "int32",
+      "type": "integer"
+    },
+    "PollingInterval": {
+      "description": "Interval to check each event source in seconds. Defaults to 30s",
+      "format": "int32",
+      "type": "integer"
+    },
+    "JobScaleRule": {
+      "description": "Scaling rule.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Scale Rule Name",
+          "type": "string"
+        },
+        "type": {
+          "description": "Type of the scale rule\neg: azure-servicebus, redis etc.",
+          "type": "string"
+        },
+        "metadata": {
+          "description": "Metadata properties to describe the scale rule.",
+          "type": "object"
```

Comment:
>object [](http://example.com/codeflow?start=19&length=6)

Why using free-form object? Free-form objects are not allowed. Please use schema for the metadata.